### PR TITLE
Pygenometracks

### PR DESCRIPTION
**Pull request by joachimwolff**
I took the wrapper from hicPlotTADs and added the `multiple` function. Any new feature that I missed? @fidelram @vivekbhr

### CODE REVIEW

**Comment by fidelram** [comment]
What is multiple? That you can overlay tracks?

Here is a list of relevant changes in the latest version:

> - Allows a track be plotted over the previous. There is not limit on the tracks that can be plotted on top of each other
> - Added option to plot links as 'triangles'. This is useful to overlay links with Hi-C data to identify the contact pixel.
> - Added line width to bed track
> - Added 'bed graph' track
> - Added a track that can plot .narrowPeaks

**Comment by joachimwolff** [comment]
Multiple is to plot n tracks with the same settings with just one command for one image.

**Comment by bgruening** [comment]
.shed.yml file is missing to get the tests running.

**Comment by vivekbhr** [comment]
@joachimwolff is this pulled from master branch of pyGenomeTracks? Some of the descriptions still mention hicPlotTADs.

**Comment by joachimwolff** [comment]
@vivekbhr It is a renamed version of the hicPlotTADs wrapper from HiCExplorer and I removed all not used code from the macros.xml file. Moreover I extended the wrapper a bit by adding the `multiple` parameter to the tracks. With this it is possible to plot n tracks with the identical settings at once and users do not have to add n tracks manually. It is a Galaxy wrapper parameter, we just loop over these files and add n tracks in the background. I am using the latest conda version of pyGenomeTracks which is 2.0. 

@bgruening I always forget this file.. :disappointed:

File: tools/pygenometracks/pyGenomeTracks.xml [hunk 1]
```diff
@@ -0,0 +1,742 @@
...
+            <param name="image_file_format" value="png" />
+            <output name="outFileName" file="master_TADs_plot.png" ftype="png" compare="sim_size" delta="35000" />
+        </test>
+    </tests>
+    <help><![CDATA[
+
+pyGenomeTracks
+==============
+
+**pyGenomeTracks** is a visualization tool to plot the topologically associating domains (TADs) in a given region.
```
Comment (bgruening) [inline tools/pygenometracks/pyGenomeTracks.xml]: this needs also to be adjusted, this tool should plot everything, TADs is just one feature of many, and it should be possible to plot tracks without having a TAD track.

File: tools/pygenometracks/pyGenomeTracks.xml [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+            <repeat name="tracks">
+                <conditional name="track_file_style_conditional">
+                    <param name="track_file_style_selector" value="vlines_track_option" />
+                    <param name="track_input_bed_single" value="domains.bed" ftype="bed" />
+                    <param name="type" value="vlines" />
+                </conditional>
+            </repeat>
+            <param name="image_file_format" value="png" />
+            <output name="outFileName" file="master_TADs_plot.png" ftype="png" compare="sim_size" delta="35000" />
+        </test>
```
Comment (bgruening) [inline tools/pygenometracks/pyGenomeTracks.xml]: a second test, without HIC would be nice.

File: tools/pygenometracks/pyGenomeTracks.xml (hunk 1 above)
Comment (joachimwolff) [reply tools/pygenometracks/pyGenomeTracks.xml]: I changed the description a bit to cover this.

**Comment by joachimwolff** [comment]
Any idea why travis quits after 20 mins because of no output from the test cases?

**Comment by mvdbeek** [comment]
Could just take too much time to download and unpack the dependency. I've restarted it now, maybe it'll pass

**Comment by bgruening** [comment]
Finally, the conda resolver had problems again. Fixed it upstream.

**Pull request merged**